mock/py/mockbuild/plugins/package_state.py

```diff
@@ -43,7 +43,7 @@ def __init__(self, plugins, conf, buildroot):
         self.inst_done = False
         self.online = self.buildroot.config['online']
         plugins.add_hook("postyum", self._availablePostYumHook)
-        plugins.add_hook("prebuild", self._installedPreBuildHook)
```

Comment:
Although this hook is not documented, it is used by several plugins. I believe it should not be removed.

Reply:
The hooks in this "prebuild" namespace are still called, this particular hook is just moved to a later timing...

Reply:
Right. I was confused and thought you are replacing the hook itself. Sorry for the noise.